web/src/agama.ts

```diff
@@ -104,8 +104,17 @@ const agama = {
    * @param options passed to the Intl.ListFormat constructor
    */
   formatList: (list: string[], options: object): string => {
-    const formatter = new Intl.ListFormat(agama.language, options);
-    return formatter.format(list);
+    try {
+      const formatter = new Intl.ListFormat(agama.language, options);
+      return formatter.format(list);
+    } catch (e) {
+      // use a trivial formatting with commas when the locale formatting fails for whatever reason
+      console.warn(
+        `Using fallback list formatting function for language "${agama.language}", details:`,
+        e,
+      );
+      return list.join(", ");
```

Comment:
Why not use the `Intl.ListFormat` for `en-us`? The result would be most probably the same, but it still uses the same function.

Reply:
OK, fair enough, although I do not think that's a problem at all. It is just a fallback for corner cases.

Reply:
But it will be add ", and" at the end. Not sure if it is what we are looking for.

Reply:
OK, but it might fail for another reason, like if you by mistake pass integers instead of strings. Then it would fail again, that `.join(", ")` handles that fine:

![image](https://github.com/user-attachments/assets/0b7d2ac7-7404-4d6e-bcbc-d154fbea7cd8)

Anyway this is a corner case and hopefully we won't get any exceptions now after dropping "zh_Hans" and using the dash separator everywhere. This is just a safety belt preventing the worst (a crash).

Reply:
If you pass integers, it will fail in any language :-)

Reply:
OK, I see. Well, let's keep the join.

Reply:
As commented in IRC, please note that Agama `formatList` only accepts a list of strings :)